vnext/Desktop.ABITests/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-	<package id="Microsoft.googletest.v140.windesktop.msvcstl.static.rt-dyn" version="1.8.1" targetFramework="native" />
+	<package id="Microsoft.googletest.v140.windesktop.msvcstl.static.rt-dyn" version="1.8.1.3" targetFramework="native" />
```

Comment:
>1.8.1.3 [](start = 83, length = 7)

Would it make sense to update the GTest as a part of a different PR?

Reply:
Yes, though it shouldn't affect. At all.
Added this change because VS complained about unconsolidated dependencies.